src/lib/structures/JoshProvider.ts

```diff
@@ -154,48 +195,59 @@ export abstract class JoshProvider<StoredValue = unknown> {
   public abstract [Method.Find](payload: Payloads.Find<StoredValue>): Awaitable<Payloads.Find<StoredValue>>;
 
   /**
+   * Get a value using a key and/or path.
    * @since 2.0.0
    * @param payload The payload sent by this provider's {@link Josh} instance.
    * @returns The payload (modified), originally sent by this provider's {@link Josh} instance.
    */
   public abstract [Method.Get]<Value = StoredValue>(payload: Payloads.Get<Value>): Awaitable<Payloads.Get<Value>>;
 
   /**
+   * Gets all data from the provider.
    * @since 2.0.0
    * @param payload The payload sent by this provider's {@link Josh} instance.
    * @returns The payload (modified), originally sent by this provider's {@link Josh} instance.
    */
   public abstract [Method.GetAll](payload: Payloads.GetAll<StoredValue>): Awaitable<Payloads.GetAll<StoredValue>>;
 
   /**
+   * Gets multiple keys from the provider.
```

Comment:
"The provider" is repetitive but if you want it, it needs to be in every description. It also might not be true? Cache would mean the data comes from somewhere else

Reply:
What cache?

Reply:
Middleware*